sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -786,6 +786,7 @@ SIREPO.app.directive('fieldEditor', function(appState, keypressService, panelSta
         controller: function($scope, $element) {
             $scope.appState = appState;
             $scope.utilities = utilities;
+            $scope.UTILS = SIREPO.UTILS;
```

Comment:
When I need some utility function that I think is of general interest, I (now) put it in the `SirepoUtils` class instead of the `utilities` angular service. It needs to be here because this is the scope referenced in the URL field definition. It's named that way as an analog to `$scope.utilities`, but it can be `SIREPO_UTILS` or whatever works best